test/integration/apiserver/apiserver_test.go

```diff
@@ -1957,16 +1963,18 @@ func TestTransform(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			rv, _ := strconv.Atoi(obj.GetResourceVersion())
-			if rv < 1 {
-				rv = 1
+			var rv string
+			if obj.GetResourceVersion() == "" {
+				rv = "0"
+			} else {
+				rv = previousRV
```

Comment:
do you trust here that there have been other (mutating) requests since previousRV was taken?

Reply:
Did you mean to ask that there have _not_ been other mutation requests? Yes, we can, since nothing but the test touches this integration etcd.

Reply:
was reading that previous is different. Maybe wrong assumption.